C2CRIBuildDir/projects/C2C-RI/src/RIGUI/src/org/fhwa/c2cri/testmodel/RITestEngine.java

```diff
@@ -372,15 +372,14 @@ public RITestEngine(TestConfiguration testConfig, TestCaseResults tcResults, Str
             RIEmulation.getInstance().setEmulationEnabled(false);            
         }
 		
-		RILogging.setNewAppender();
         logger.configureLogging(testName, testConfigName, testDescription, checksum, emulationEnabled, reinitializeEmulation);
 
         // Remove any invalid XML characters that may exist.
         // ref : http://www.w3.org/TR/REC-xml/#charsets
         // jdk 7
         Pattern xmlInvalidChars =
          Pattern.compile("[^\\u0009\\u000A\\u000D\\u0020-\\uD7FF\\uE000-\\uFFFD\\x{10000}-\\x{10FFFF}]");
-        logger.logEvent(RITestEngine.class.getName(), RILogging.RI_INIT_EVENT, xmlInvalidChars.matcher(testConfig.to_LogFormat()).replaceAll(""));
+        logger.logEvent("C2CRIDebug", RILogging.RI_INIT_EVENT, xmlInvalidChars.matcher(testConfig.to_LogFormat()).replaceAll(""));
```

Comment:
Why do we replace the `RITestEngine `class with `C2CRIDebug`?

Reply:
It has to do with how log4j v1 was configured. The class names would inherit the configuration of the package, namely "org.fhwa.c2cri" and "net.sf.jameleon". I explicitly defined "C2CRIDebug" in the log4j configuration so it was more obvious which log file statements would be written to. Basically the Root logger can be used to write to the standout output of the C2CRIOutput.log file (which is used anytime we get a Logger with a class name), and for the messages that were going to the C2CRIDebug.log we can get the Logger with "C2CRIDebug"